stdlib/public/runtime/MetadataImpl.h

```diff
@@ -188,6 +189,7 @@ template <class Impl, class T> struct RetainableBoxBase {
   static constexpr size_t stride = sizeof(T);
   static constexpr bool isPOD = false;
   static constexpr bool isBitwiseTakable = true;
+  static const bool isAtomic;
```

Comment:
Fixed in https://github.com/apple/swift/pull/7557/commits/dfbdf00647d8e3b5c9830cb2bc6b8fbbaa06320b